readthedocs/core/static-src/core/js/doc-embed/search.js

```diff
@@ -275,23 +296,27 @@ function attach_elastic_search_query_mkdocs(data) {
                 var results = data.results || [];
 
                 if (results.length) {
-                    var searchResults = $('#mkdocs-search-results');
-                    searchResults.empty();
+                    let searchResults = document.getElementById('mkdocs-search-results');
```

Comment:
We could also just kill the mkdocs integration, isn't currently being used (only on my test project).

Reply:
I'm 👍🏼 on removing things we are not using 😄

Reply:
Perhaps this is something that we instead address with generic version of our search integration. I'm :+1: in both directions, if we aren't using the code it shouldn't hurt to keep it around either, but also happy to see unused code disappeared.

Reply:
I think we should remove it, and work towards shipping our livesearch as a generic JS library. The less code we have to maintain the better. 👍